features/conic-gradients.yml.dist

```diff
@@ -2,13 +2,10 @@
 # Do not edit this file by hand. Edit the source file instead!
 
 status:
-  baseline: high
-  baseline_low_date: 2020-11-17
-  baseline_high_date: 2023-05-17
+  baseline: false
```

Comment:
This will be fixed by https://github.com/mdn/browser-compat-data/pull/23603.

Reply:
Should we site on this, until it's closer to the release? Or should we just avoid doing a release until this is fixed upstream?

Reply:
Let's wait for the next BCD release before merging this PR.